roles/ceilometer-common/tasks/main.yml

```diff
@@ -14,7 +14,7 @@
         owner=ceilometer group=ceilometer
 
 - name: ceilometer config
-  template: src=etc/ceilometer/{{ item }} dest=/etc/ceilometer/{{ item }} mode=0644
+  template: src=etc/ceilometer/{{ item }} dest=/etc/ceilometer/{{ item }} mode={{ 0644 if 'policy.json' in item else 0640 }} owner=ceilometer group=ceilometer
```

Comment:
Policy files are read by horizon as well as the regular service. All the other services do the policy file in this way.